docs/docs/0.3.2/documentation/README.md

```diff
@@ -390,10 +390,10 @@ spec:
       ...
 ```
 
-::: tip
-When integrating [Kong for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller) with Kuma you have to annotate every `Service` that you want to pass traffic to with [`ingress.kubernetes.io/service-upstream=true`](https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/references/annotations.md#ingresskubernetesioservice-upstream) annotation.
-Otherwise Kong will do the load balancing which unables Kuma to do the load balancing and apply policies. 
-:::
+The optimal gateway in Kubernetes mode would be Kong. You can use [Kong for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller) to implement authentication, transformations, and other functionalities across Kubernetes clusters with zero downtime. When integrating [Kong for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller) with Kuma you have to annotate every `Service` that you want to pass traffic to with [`ingress.kubernetes.io/service-upstream=true`](https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/references/annotations.md#ingresskubernetesioservice-upstream) annotation. Otherwise Kong will do the load balancing which enables Kuma to do the load balancing and apply policies. 
+
+For an in-depth example on deploying Kuma with [Kong for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller), please follow this [demo application guide](https://github.com/Kong/kuma-demo/tree/master/kubernetes#31-kong-gateway-integration).
```

Comment:
Link to `#31-kong-gateway-integration` is too fragile (because of `#31` which is about change at any moment)

Reply:
Im currently updating the kubernetes docs to be more like vagrant where we have sections by policies.

removed for meantime